### PR TITLE
update discovery rule with connection to taxonomy

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

**Comment by orrabin** [comment]
DO NOT MERGE before https://github.com/theforeman/foreman_discovery/pull/172 is merged.

@elobato edit - it's merged now

**Comment by lzap** [comment]
Can you write a paragraph about what happens when you have a discovered host with a different organization than rule belongs to?

File: plugins/foreman_discovery/3.0/index.md [hunk 1]
```diff
@@ -458,6 +460,11 @@ By default, Foreman does not trigger auto discovery automatically. This must
 be explicitly turned on in
 *Administer > Settings > Discovered > discovery_auto*.
 
+When a discovered host has an organization or location that is different from a rule
+that rule can not be used to auto provision the host. When troubleshooting why a rule that
+was expected to be used to auto provision a host was disregarded please check
+the organization/location of both the host and the rule.
```
Comment (orrabin) [inline plugins/foreman_discovery/3.0/index.md]: @lzap I'm not sure this is the best place for it, what do you think?

Comment (lzap) [inline plugins/foreman_discovery/3.0/index.md]: > @lzap I'm not sure this is the best place for it, what do you think?

It's fine.

## 

Later,
 Lukas #lzap Zapletal

File: plugins/foreman_discovery/3.0/index.md [hunk 2]
```diff
@@ -443,13 +443,15 @@ create such a rule:
...
-  rule. If a limit was reached, the rule will not tkae effect until one or
+  rule. If a limit was reached, the rule will not take effect until one or
   more hosts are deleted. Typical use case are rules per server rack or row
   when it is necessary to change provisioning parameters like hostname or host
   group per each entry.
 * **Priority** puts the rules in order. Must be greater than zero and low
   numbers go first. Rules are always matched by priority given.
 * **Enabled** flag is used for temporary shutdown of rules.
+* **Organizations\Locations** the rule can be used only on discovered hosts
+ that is connects to the same organization and location as the rule.
```
Comment (dLobatog) [inline plugins/foreman_discovery/3.0/index.md]: - that are connected

**Comment by dLobatog** [comment]
@orrabin I find the description a bit dense to understand, the more we write the less they read... perhaps something along the lines of 
`discovered hosts can only auto provision from rules in the same organization/location as the discovered host` 
is clearer..?

It might just be me though.

**Comment by orrabin** [comment]
@lzap it looks to me that @eLobato's explanation can be put here: https://github.com/theforeman/theforeman.org/pull/356/files#diff-ba595c13464f36ac43924cc97331a4e0R453 and then just skip the paragraph.
Do you have any objections?

**Comment by domcleal** [comment]
@orrabin good suggestion

**Comment by lzap** [comment]
Yeah :+1:

**Pull request merged**